### DISCUSSION TITLE
InteractiveRectangle - Interactive Area positioning problem

### DISCUSSION THREAD

**Discussion by davidmaruscsak00** [cables-gl/cables_docs#230] [Answers & Questions]
Hello everyone,

I am working on a patch in cables, where I use InteractiveRectangle node for creating interactive areas for mouse click.

I am using a ForceCanvasSize node to make sure that my patch is always 1920x1080 px resolution, regardless the viewport size.
I have Center in Parent and Scale to Fit Parent checked in that node.

When I check these to options my patch looks great on every screen, however the Bounding Box of the Interactive Rectangle doesn't scale and it is misaligned.

When I check the Render Rectangle option. The rectangle of the InteractiveArea node is in a correct position only the BoundingBox is misaligned.

I have attached a picture to show the difference between the RenderRectangle and the BoundingBox positions.

Also you can find my patch here:
https://cables.gl/p/IrjD_1

![279143163_1410670326040607_2287110097431185133_n](https://user-images.githubusercontent.com/23195286/166143710-c9dd34cb-4b2a-4b6c-955b-e344a083f2a9.png)

**Comment by steam0r** [comment]
did you find a solution for this already? looks fine to me on mac retina and non retina display:

<img width="583" alt="Bildschirmfoto 2022-05-04 um 09 29 17" src="https://user-images.githubusercontent.com/151271/166638844-7e4113ba-ee0b-4574-90b8-ee38ee30c6cf.png">
